dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -3710,6 +3710,24 @@ public void refresh(ContentType type) throws DotReindexStateException {
 
     }
 
+    @WrapInTransaction
+    @Override
+    public Contentlet saveContentOnVariant(final Contentlet contentlet, final String variantName,
```

Comment:
who is using this thing, why it is being used at this level?
I think this could be added on SaveActionlet and SaveDraft rather than the API

Reply:
really I think it is a matter of name 

https://github.com/dotCMS/core/pull/24450/commits/39c844389237f85a99ed82dc502d5018e20588a4#diff-fa1ceaa19618a6b2bbc30e24c6f930b4971f417db50babb748c2e2837ba9eb82R3714